util/rbac/rbac.go

```diff
@@ -97,6 +97,16 @@ func (e *Enforcer) EnforceErr(rvals ...interface{}) error {
 			for i, rval := range rvals[1:] {
 				rvalsStrs[i] = fmt.Sprintf("%s", rval)
 			}
+			switch s := rvals[0].(type) {
+			case jwt.Claims:
+				claims := s.(*jwt.StandardClaims)
```

Comment:
First value of `rvals` is not necessary `*jwt.StandardClaims`. One unit test failed because of it: https://github.com/argoproj/argo-cd/pull/3850/checks?check_run_id=812569880

I would suggest to use `jwt.MapClaims` method to convert `rvals[0]` to Map claims and then extract `sub` and `iat` fields. Please also add a check to handle edge cases. E.g. `rval` is empty slice or claims don't have `sub`  or `iat` fields.

https://github.com/argoproj/argo-cd/blob/8aadc310c94e52ac13b6f690c5a2f703c87bc614/util/jwt/jwt.go#L11

Reply:
Already addressed, thanks Alex!